spec/ruby/core/method/super_method_spec.rb

```diff
@@ -22,4 +22,18 @@
     sss_meth.receiver.should == obj
     sss_meth.name.should == :overridden
   end
+
+  it "returns nil when there's no super method in the parent" do
+    method = Object.new.method(:method)
+    method.super_method.should == nil
+  end
+
+  it "doesn't look for removed methods" do
```

Comment:
I'd change this to `it 'returns nil when the method is removed'` or something along those lines. "doesn't look for..." doesn't really indicate that `super_method` returns in this case.